### PR TITLE
Doc: Prepare for 14.0-beta2 release

### PR DESCRIPTION
**Pull request by TrueBrain**
## Motivation / Problem

Two critical bugs spoiling all the fun.

## Description

Changelog for a beta2, which includes fixes for these bugs.

## Limitations

Not really in the mood to go through another release cycle, so please amend this PR where you see issues. I just automated creating changelogs, and this is the result of that automation. It hasn't been touched otherwise :)

## Checklist for review

Some things are not automated, and forgotten often. This list is a reminder for the reviewers.
* The bug fix is important enough to be backported? (label: 'backport requested')
* This PR touches english.txt or translations? Check the [guidelines](https://github.com/OpenTTD/OpenTTD/blob/master/docs/eints.md)
* This PR affects the save game format? (label 'savegame upgrade')
* This PR affects the GS/AI API? (label 'needs review: Script API')
    * ai_changelog.hpp, game_changelog.hpp need updating.
    * The compatibility wrappers (compat_*.nut) need updating.
* This PR affects the NewGRF API? (label 'needs review: NewGRF')
    * newgrf_debug_data.h may need updating.
    * [PR must be added to API tracker](https://wiki.openttd.org/en/Development/NewGRF/Specification%20Status)

### CODE REVIEW

File: changelog.txt [hunk 1]
```diff
@@ -1,3 +1,12 @@
+14.0-beta2 (2023-02-04)
+------------------------------------------------------------------------
+Change: Decouple and remove landscape-dependent cargo types (#11719)
```
Comment (LordAro) [inline changelog.txt]: Doesn't really describe what the implications of the change are

File: changelog.txt [hunk 2]
```diff
@@ -1,3 +1,12 @@
...
+Fix #11972: Incorrect minimum width allocated to graph (#11974)
```
Comment (LordAro) [inline changelog.txt]: Which graph?

**Review by LordAro — COMMENTED** [review]
No mention of the signal on non-rail crash?

File: changelog.txt [hunk 3]
```diff
@@ -1,3 +1,14 @@
+14.0-beta2 (2023-02-04)
+------------------------------------------------------------------------
+Change: Decouple and remove landscape-dependent cargo types (#11719)
```
Comment (michicc) [inline changelog.txt]: Maybe this?

```suggestion
Change: [NewGRF] Improved support for redefining default cargo types (#11719)
Fix #11802: Made determining water region edge traversability more robust (#11986)
```

Comment (michicc) [reply changelog.txt]: Kinda abusing suggestions here :)

File: changelog.txt [hunk 4]
```diff
@@ -1,6 +1,7 @@
 14.0-beta2 (2023-02-04)
 ------------------------------------------------------------------------
-Change: Decouple and remove landscape-dependent cargo types (#11719)
+Change: [NewGRF] Improved support for redefining default cargo types (#11719)
+Fix #11802: Made determining water region edge traversability more robust (#11986)
```
Comment (TrueBrain) [inline changelog.txt]: My OCD doesn't appreciate this sorting :P :P

File: changelog.txt [hunk 5]
```diff
@@ -1,3 +1,15 @@
+14.0-beta2 (2023-02-04)
+------------------------------------------------------------------------
+Change: [NewGRF] Improved support for redefining default cargo types (#11719)
+Fix #11802: Made determining water region edge traversability more robust (#11986)
+Fix #11975: Inconsistent behaviour when changing first AI company settings (#11976)
+Fix #11972: Year cut off in graph windows (#11974)
+Fix #11968: Crash when opening orders window of new vehicles (#11973)
+Fix #11966: Monospace text in windows may not have been fully scrollable (#11981)
+Fix #11982: Crash when trying to place signals on things other than plain rails (#11977)
+Fix: Second colour vehicle-type default liveries were not being updated (#11971)
```
Comment (michicc) [inline changelog.txt]:
```suggestion
Change: [NewGRF] Improved support for redefining default cargo types (#11719)
Fix #11982: Crash when trying to place signals on things other than plain rails (#11977)
Fix #11975: Inconsistent behaviour when changing first AI company settings (#11976)
Fix #11972: Year cut off in graph windows (#11974)
Fix #11968: Crash when opening orders window of new vehicles (#11973)
Fix #11966: Monospace text in windows may not have been fully scrollable (#11981)
Fix #11802: Made determining water region edge traversability more robust (#11986)
Fix: Second colour vehicle-type default liveries were not being updated (#11971)
```

File: changelog.txt [hunk 6]
```diff
@@ -1,3 +1,15 @@
+14.0-beta2 (2023-02-04)
+------------------------------------------------------------------------
+Change: [NewGRF] Improved support for redefining default cargo types (#11719)
+Fix #11975: Inconsistent behaviour when changing first AI company settings (#11976)
+Fix #11972: Year cut off in graph windows (#11974)
+Fix #11968: Crash when opening orders window of new vehicles (#11973)
+Fix #11966: Monospace text in windows may not have been fully scrollable (#11981)
+Fix #11982: Crash when trying to place signals on things other than plain rails (#11977)
+Fix #11802: Made determining water region edge traversability more robust (#11986)
+Fix: Second colour vehicle-type default liveries were not being updated (#11971)
```
Comment (michicc) [inline changelog.txt]:
```suggestion
Change: [NewGRF] Improved support for redefining default cargo types (#11719)
Fix #11982: Crash when trying to place signals on things other than plain rails (#11977)
Fix #11975: Inconsistent behaviour when changing first AI company settings (#11976)
Fix #11972: Year cut off in graph windows (#11974)
Fix #11968: Crash when opening orders window of new vehicles (#11973)
Fix #11966: Monospace text in windows may not have been fully scrollable (#11981)
Fix #11802: Made determining water region edge traversability more robust (#11986)
Fix: Second colour vehicle-type default liveries were not being updated (#11971)
```

**Review by PeterN — APPROVED** [review]

**Pull request merged**